### DISCUSSION TITLE
About running starrocks in k8s cluster

### DISCUSSION THREAD

**Discussion by clcc2019** [StarRocks/starrocks#3937] [Announcements]
In the face of product containerization, the service runs in the k8s cluster, and starrocks currently does not have a containerization solution. How to maintain the IP addresses of the fe and be nodes? Because the ip address changes in the cluster

**Comment by harryhan1989** [comment]
[starrokcs docker comtainers](https://hub.docker.com/repository/docker/harryhan1989/starrokcs)
我在测试使用starrocks解决方案的时候也碰到类似的问题，所以直接把社区版各版本都编译了docker镜像，可在k8s或者其他容器环境中使用，具体用法，上面有说明文档，编译dockerfile源码托管在github上，有兴趣自己编译。如果有新版本发我，我没来得及编译发布，可以在github上给我提。
